### PR TITLE
Make sure apparmor profile is added only for Unbuntu 24 as it is only…

### PR DESCRIPTION
**Pull request by adityatoshniwal**
… one creating problem.w

### CODE REVIEW

File: pkg/debian/build.sh [hunk 1]
```diff
@@ -68,10 +68,28 @@ fakeroot dpkg-deb --build "${SERVERROOT}" "${DISTROOT}/${APP_NAME}-server_${APP_
 echo "Creating the desktop package..."
 mkdir "${DESKTOPROOT}/DEBIAN"
 
-cat << EOF > "${DESKTOPROOT}/DEBIAN/conffiles"
+# Ubuntu 24 requires apparmor profile to work.
+UBUNTU_CODENAME=$(grep "^UBUNTU_CODENAME=" /etc/os-release | awk -F "=" '{ print $2 }')
+if [ "${UBUNTU_CODENAME}" == 'noble' ]; then
```
Comment (dpage) [inline pkg/debian/build.sh:73]: Hard coded on a single codename? That means we need to change the code with every release. Please change to a numeric comparison on the version number.

File: pkg/debian/build.sh [hunk 2]
```diff
@@ -68,10 +68,28 @@ fakeroot dpkg-deb --build "${SERVERROOT}" "${DISTROOT}/${APP_NAME}-server_${APP_
...
+  mkdir -p "${DESKTOPROOT}/etc/apparmor.d"
+  cp "${SOURCEDIR}/pkg/debian/pgadmin4-aa-profile" "${DESKTOPROOT}/etc/apparmor.d/pgadmin4"
+
+  cat << EOF > "${DESKTOPROOT}/DEBIAN/postinst"
+#!/bin/sh
+
+STATUS="$(systemctl is-active apparmor.service)"
+if [ "${STATUS}" == "active" ]; then
+    echo "Restarting apparmor.service to load pgAdmin profile..."
+    systemctl restart apparmor.service
```
Comment (dpage) [inline pkg/debian/build.sh:87]: Do we need to restart it, or can we just reload it?

File: web/pgadmin/about/__init__.py [hunk 3]
```diff
@@ -123,6 +123,13 @@ def detect_browser(request):
     agent = request.environ.get('HTTP_USER_AGENT')
     os_details = parse(platform.platform()).ua_string
 
+    try:
```
Comment (dpage) [inline web/pgadmin/about/__init__.py:126]: How is this related to AppArmor?

File: pkg/debian/build.sh (hunk 1 above)
Comment (dpage) [inline pkg/debian/build.sh:73]: Hard coded on a single codename? That means we need to change the code with every release. Please change to a numeric comparison on the version number.

**Review by dpage — CHANGES_REQUESTED** [review]

File: web/pgadmin/about/__init__.py (hunk 3 above)
Comment (adityatoshniwal) [reply web/pgadmin/about/__init__.py:126]: It is not, I found it helpful to add in the about dialog.

File: pkg/debian/build.sh (hunk 2 above)
Comment (adityatoshniwal) [reply pkg/debian/build.sh:87]: OK I found a better way. This will only load pgadmin4 
```
apparmor_parser -r /etc/apparmor.d/pgadmin4
```

File: web/pgadmin/about/__init__.py (hunk 3 above)
Comment (dpage) [reply web/pgadmin/about/__init__.py:125]: It should be in a separate PR then.

**Review by dpage — CHANGES_REQUESTED** [review]

File: web/pgadmin/about/__init__.py (hunk 3 above)
Comment (adityatoshniwal) [reply web/pgadmin/about/__init__.py:125]: OK. I'll remove the change.

**Review by dpage — APPROVED** [review]
